Lombiq.Hosting.Azure.ApplicationInsights/TelemetryInitializers/IgnoreFailureInitializer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Lombiq.Hosting.Azure.ApplicationInsights.Extensions;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.Extensibility.Implementation;
+using System;
+
+namespace Lombiq.Hosting.Azure.ApplicationInsights.TelemetryInitializers;
+
+public class IgnoreFailureInitializer : ITelemetryInitializer
+{
+    private readonly IServiceProvider _serviceProvider;
+
+    public IgnoreFailureInitializer(IServiceProvider serviceProvider) =>
+        _serviceProvider = serviceProvider;
+
+    public void Initialize(ITelemetry telemetry)
+    {
+        var operationTelemetry = telemetry as OperationTelemetry;
+        if (operationTelemetry?.Success != false) return;
```

Comment:
```suggestion
        if (operationTelemetry?.Success == true) return;
```

Reply:
If it is not an `OperationTelemetry`, then it will continue with `operationTelemetry?.Success == true`  instead of the expected return.

Reply:
I see, indeed.